RUFAS/routines/animal/ration/ration_config.py

```diff
@@ -176,6 +177,8 @@ def __init__(
             RUP degradability in each feed (% of RUP).
         limit__list : list, optional
             Limiting upper bounds for each feed (kg).
+        feed_minimum_list : list, optional
```

Comment:
```suggestion
        feed_minimum__list : list[float], default []
```
I think it would be better to not refer to this argument as "optional" because this term in Python is a little ambiguous. "optional" can mean that `None` is an acceptable value for this parameter, or it can mean that the caller function doesn't have to specify a value for this argument. In this case, it is only the latter and I think this is indicated more explicitly by saying what the default is.

Reply:
Excellent suggestion!